data/AZ/incentive_relationships.json

```diff
@@ -1,12 +1,44 @@
 {
   "exclusions": {
-    "AZ-79": ["AZ-80"],
-    "AZ-81": ["AZ-82"],
-    "AZ-83": ["AZ-84"],
-    "AZ-85": ["AZ-86"],
-    "AZ-87": ["AZ-88"],
-    "AZ-89": ["AZ-90"],
-    "AZ-91": ["AZ-92"],
-    "AZ-93": ["AZ-94"]
+    "AZ-79": {
+      "ids": [
+        "AZ-80"
+      ]
```

Comment:
more verbose/cluttered at this point, but good to be explicit anyway

Reply:
We shouldn't be hand-editing these anymore anyway, and ideally won't even really be looking closely at them.